templates/controllers/tab/settings/appearance/form/appearanceForm.tpl

```diff
@@ -114,6 +114,8 @@
 	{/fbvFormArea}
 	{* end Layout *}
 
+	{$thumbnailsSizeSettings}
```

Comment:
Calling the variables this way is still OMP-specific. Better to use a generic name that could be used for different purposes, including (but no limited to) the thumbnail stuff in OMP.